app/models/mobile_network.rb

```diff
@@ -14,6 +14,9 @@ def self.participating_in_pilot
     participating
   end
 
+  scope :excluded_fe_networks, -> { where(brand: %w[giffgaff O2 Three SMARTY]) }
```

Comment:
This feels a bit icky, I think it would've been better as a flag/data item on the model?

Reply:
sure, this works so i'll merge this now and make the change once everything is sorted.